cmd/primateRecon/primateRecon.go

```diff
@@ -130,7 +160,7 @@ func primateReconMle(inFastaFilename string, inTreeFilename string, humanBias bo
 		log.Fatalf("Didn't find hca in the tree\n")
 	}
 
-	if !humanBias {
+	if !(humanBias || chimpBias) {
 		humanAltNode = expandedTree.FindNodeName(tree, "hg38alt")
```

Comment:
Probably not needed, but we could think of doing something similar with variation we observe in chimps

Reply:
Is this saying that if there is no humanBias or chimpBias, then the uncertain base in the HCA will become the hg38Alt? Is it biologically true that hg38Alt is not as good at predicting HCA as human or chimp Ref alleles?

Reply:
This is adding an additional node to the tree that is a "second human genome."  In the case of an A/G SNP in humans, we can make two human genomes (where one is A and one is G) so that we don't arbitrarily bias towards one of the alleles for a segregating site.

Reply:
OK I see!